---
title: MSSQLSERVER_8630 | Документация Майкрософт
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.component: errors-events
ms.reviewer: ''
ms.suite: sql
ms.technology: supportability
ms.tgt_pltfrm: ''
ms.topic: language-reference
helpviewer_keywords:
- 8630 (Database Engine error)
ms.assetid: 45a9d084-b5ad-435b-ac19-5f4352515817
caps.latest.revision: 14
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: e4d2c566a6d4620b53154477bffd4737658cfa11
ms.sourcegitcommit: f1caaa156db2b16e817e0a3884394e7b30fb642f
ms.translationtype: HT
ms.contentlocale: ru-RU
ms.lasthandoff: 05/04/2018
---
# <a name="mssqlserver8630"></a>MSSQLSERVER_8630
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|8630|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|EXECUTION_ERR|  
|Текст сообщения|внутренняя ошибка обработчика запросов: обработчик запросов обнаружил непредвиденную ошибку во время выполнения.|  
  
## <a name="explanation"></a>Объяснение  
Внутренняя ошибка обработчика запросов.  
  
## <a name="user-action"></a>Действие пользователя  
Обратитесь к основному поставщику услуг. Предоставьте информацию из этого сообщения об ошибке.  
  
